### ISSUE TITLE
Create a failure technique for Non-text contrast based on a lightbox effect

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#3203]
This need was identified in discussions on Focus Not Obscured and whether a lightbox effect would be considered to partially obscure the item receiving keyboard focus.

At some point the wording of Focus Not Obscured (minimum and enhanced) ditched the word and definition for "obscured" as part of its normative text, and now uses "hidden"; however, the discussion on how a lightbox effect could still lead to a failure of sufficient contrast for the focus indicator should be captured.

Essentially: if a lightbox effect is used on a dialog, and focus is not constrained to the dialog, a keyboard user may be able to tab out of the dialog, resulting in the item with focus being in the 'greyed' part of the screen (the lightbox effect). At that point, the contrast between the focus indicator and the background must still achieve 3:1 or it will fail Non-text Contrast. At some point it will also fail Focus Appearance as well.

**Comment by junykallukalam** [comment]
I tried it out in my codepen, https://codepen.io/Juny-Kallukalam/pen/LYgMWNR

**Comment by davidc-gds** [comment]
A WCAG 2.x group took a look at this issue today and noted that it seems like @mbgower is doing some work behind the scenes related to 'Non-text contrast' and the lightbox effect.

We'll hold off on looking at this further until that work has come back around.

Perhaps that work has already been posted somewhere else in this repository, though we weren't able to locate it at this time.

**Comment by mbgower** [comment]
@dav-idc 
I incorporated the [wording](https://www.w3.org/WAI/WCAG22/Understanding/focus-not-obscured-minimum.html#:~:text=Another%20form%20of,item%20has%20focus.) into the Understanding document.

However, I have not created a failure technique. Someone else can feel free to pick this up.

**Comment by venkatvemuri28** [comment]
Raised a PR https://github.com/w3c/wcag/pull/4673
